### PR TITLE
Manager v2 polish

### PR DESCRIPTION
**Pull request by LFBarreto**
Polishes on ManagerV2:
* fixed issue when updatable apps list is opened and we click on the update all button that closed the collapse list
* manager modals layout fixed and centered
* device storage card layout polish
* empty state on installed list on white background
* dependent modal desc wording fixed
* fade in animation removed on list items
* border radius on catalog card fixed

### Type

UI Polish

### Context

ManagerV2

### Parts of the app affected / Test plan

Manager V2

### CODE REVIEW

**Review by gre — APPROVED** [review]

File: src/renderer/screens/manager/AppsList/index.js [hunk 1]
```diff
@@ -54,9 +60,17 @@ const AppsList = ({ deviceInfo, result, exec, t }: Props) => {
...
   const progress = state.currentProgress;
   const isIncomplete = isIncompleteState(filteredState);
-  const distribution = distribute(filteredState);
 
   const { installQueue, uninstallQueue, currentError } = filteredState;
+  const jobInProgress = installQueue.length > 0 || uninstallQueue.length > 0;
+
+  const distribution = useMemo(() => {
+    const newState = installQueue.length
+      ? predictOptimisticState(reducer(filteredState, { type: "install", name: installQueue[0] }))
```
Comment (gre) [inline src/renderer/screens/manager/AppsList/index.js:69]: I don't understand why we do this. can you explain and/or comment in the code?

Comment (LFBarreto) [reply src/renderer/screens/manager/AppsList/index.js:69]: For the optimistic device storage bar as you suggested.
It allows to show the size after the installs are complete

Comment (gre) [reply src/renderer/screens/manager/AppsList/index.js:69]: makes sense!

**Comment by gre** [comment]
@LFBarreto some conflict :)

**Review by Arnaud97234 — APPROVED** [review]

**Pull request merged**